.gitignore

```diff
@@ -11,4 +11,5 @@ dist-ssr
 server/dist
 public/dist
 .turbo
-test-results
\ No newline at end of file
+test-results
+.tool-versions
```

Comment:
This is used by [asdf](https://asdf-vm.com/manage/versions.html#via-tool-versions-file), the tool version manager I use. Can remove if needed.

Reply:
pls remove